syntax = "proto3";
package math_opt_benchmark;

message Item {
  int32 index = 1;
  int32 freq = 2;
  int32 aisle = 3;
}

message Substitution {
  int32 index = 1;
  repeated int32 sub_idxs = 2;
}

message Order {
  repeated int32 items = 1;
  repeated Substitution subs = 2;
}

message OrderDataset {
  repeated Order orders = 1;
  int32 nItems = 2;
}
